### PR TITLE
Update CODEOWNERS to guard local_migrations too

### PR DESCRIPTION
**Pull request by tinyels**
## Description

what the title says

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -12,3 +12,5 @@
 # Migrations are where database schema changes are made and we want them reviewed by people working on our database guidelines
 /migrations/* @chrisgilmerproj @tinyels @garrettqmartin8 @gidjin @lynzt @reggieriser @rdhariwal @Ryan-Koch
 
+# we also want to make sure the local migrations are not doing schema changes
+/Local_migrations/* @chrisgilmerproj @tinyels @garrettqmartin8 @gidjin @lynzt @reggieriser @rdhariwal @Ryan-Koch
```
Comment (reggieriser) [inline .github/CODEOWNERS]: Does case matter with this?  I'm not sure, but the directory is all lower case.

Comment (tinyels) [reply .github/CODEOWNERS]: And this is why I asked for a code review for something trivial! 🤦‍♀ Thanks @reggieriser

**Comment by chrisrcoles** [comment]
Can we make sure we get representation from all teams? I don't see anyone from Roci - please add Jim or myself to the codeowners file. Thx

**Comment by tinyels** [comment]
Thanks @chrisrcoles and @jim for joining us!

**Pull request merged**